src/Momento.Sdk/Exceptions/AlreadyExistsException.cs

```diff
@@ -1,14 +1,14 @@
 ﻿namespace Momento.Sdk.Exceptions;
 
+using System;
+
 /// <summary>
 /// Resource already exists
 /// </summary>
 public class AlreadyExistsException : SdkException
 {
-    public AlreadyExistsException(string message) : base(MomentoErrorCode.ALREADY_EXISTS_ERROR, message)
-    {
-    }
-    public AlreadyExistsException(string message, MomentoErrorTransportDetails transportDetails) : base(MomentoErrorCode.ALREADY_EXISTS_ERROR, message, transportDetails)
+    public AlreadyExistsException(string message, MomentoErrorTransportDetails? transportDetails=null, Exception? e=null) : base(MomentoErrorCode.ALREADY_EXISTS_ERROR, message, transportDetails, e)
```

Comment:
curious what prompted the change to nullable for the transport details here.  not a blocker.

Reply:
There's a fallthrough case in `CacheExceptionMapper` where we don't have transport details to add (`UnknownException`).

Reply:
but wouldn't that only impact the signature for the `UnknownException` constructor, and not these other ones?